.github/workflows/action-on-PR-labeled.yml

```diff
@@ -25,6 +40,35 @@ jobs:
               repo: context.repo.repo,
               body: `It looks like your PR is a breaking change, but you didn't provide a migration guide.
```

Comment:
```suggestion
              body: `It looks like your PR is a breaking change, but **you didn't provide a migration guide**.
```

A little bit louder so it's easier to scan? (Same for the release note part)

<img width="600" alt="image" src="https://github.com/user-attachments/assets/efe0bf66-970a-42af-852f-508377c84a34" />

Reply:
Agreed